learn/tanssi/appchain-services/block-production.md

```diff
@@ -0,0 +1,176 @@
+
+Infrastructure poses a huge challenge for developers, requiring them to bootstrap sequencers, data preservers, and RPC endpoints, while also managing integrations, interoperability, and security. This demands valuable time and resources, diverting focus from what truly matters: delivering value to their users.
+
+Tanssi orchestrates resources, allowing developers to deploy decentralized networks (also known as actively validated services or AVSs) that are fully adaptable to any specific application or use case. In Tanssi terms, these application-specific blockchains are referred to as appchains. In this analogy, the Tanssi network resembles [Kubernetes](https://kubernetes.io){target=\_blank} in its role as an orchestrator, managing resources to guarantee the liveness and performance of the appchains.
+
+The protocol also tackles the security front by allowing appchains to select and connect to external security providers (like [Symbiotic](/learn/tanssi/external-security-providers/symbiotic/){target=\_blank}), ensuring Ethereum-grade security right from the start.
+
+This article covers the necessary aspects to consider when building and deploying your own modular blockchain, along with the most relevant technical aspects of the Tanssi protocol.
+
+## Block Production as a Service {: #block-production-as-a-service }
+
+The Tanssi protocol provides block production as a service, orchestrating a decentralized and trustless set of sequencers, ensuring the appchains' liveness. To do so, the protocol bridges both ends:
+
+- **Node operators** - who run sequencers, offering their block production services to get rewards
+- **Appchains** - which require transactions to be executed and included in blocks, keeping the network's liveness
+
+The protocol assigns a subset of sequencers to provide services to each appchain, rotating them after a period of time. The sequencers can serve any appchain, regardless of the custom logic they implement. On the other hand, appchains deployed through Tanssi can customize their runtime as much as they need to fit their use case and upgrade the logic at any moment in a forkless fashion without worrying about the sequencer's setup.
+
+The following diagram illustrates how Tanssi assigns two sequencers to each active appchains, selecting them from a decentralized set of sequencers.
+
+```mermaid
+flowchart LR
+    subgraph appchain1 [Appchain 1]
+    s1bis[Sequencer 1]
+    s2bis[Sequencer 2]
+    end
+    subgraph appchain2 [Appchain 2]
+    s3bis[Sequencer 3]
+    s4bis[Sequencer 4]
+    end
+    Tanssi[
+        Tanssi Network
+        Orchestrator
+    ]
+    subgraph sequencers [Sequencers Pool]
+    direction RL
+    s1[Sequencer 1]
+    s2[Sequencer 2]
+    s3[Sequencer 3]
+    s4[Sequencer 4]
+    sn[Sequencer N]
+    end
+    sequencers-- Managed by -->Tanssi
+    Tanssi-- Assigns Sequencers -->appchain1
+    Tanssi-- Assigns Sequencers -->appchain2
```

Comment:
```suggestion
flowchart TB
    subgraph appchain1 [Appchain 1]
        s1bis[Sequencer 1]
        s2bis[Sequencer 2]
    end

    subgraph appchain2 [Appchain 2]
        s3bis[Sequencer 3]
        s4bis[Sequencer 4]
    end

    Tanssi[Tanssi Network<br/>Orchestrator]

    subgraph sequencers [Sequencers Pool]
        direction LR
        s1[Sequencer 1]
        s2[Sequencer 2]
        s3[Sequencer 3]
        s4[Sequencer 4]
        sn[Sequencer N]

        s1 --- s2 --- s3 --- s4 --- sn
    end

    sequencers -- Managed by --> Tanssi
    Tanssi -- Assigns Sequencers --> appchain1
    Tanssi -- Assigns Sequencers --> appchain2
```

Reply:
Just mainly because it renders a bit too small this way

Reply:
Much better!